docs/tfengine/README.md

```diff
@@ -6,9 +6,19 @@ The Terraform Engine is a tool to generate complete end-to-end Terraform
 deployments for Google Cloud with security, compliance, and best practices baked
 in.
 
-It introduces the concept of "templates". Templates can be used to generate
-multiple Terraform modules and configuration specific to your Google Cloud
-organization and structure.
+Use our [example](../../examples/tfengine) configs to quickly get started.
+
+## Terminology
+
+The Terraform engine introduces the concept of "templates". Templates can be
+used to generate
+[Terraform root modules](https://www.terraform.io/docs/glossary.html#root-module)
+specific to your Google Cloud organization and structure.
+
+Users typically pick from a set of
+[recipes](../../templates/tfengine/recipes) which implement a template for one
```

Comment:
I think recipes need to be described differently, they're mainly a way to group components together.

Reply:
Users don't really need to know about components I feel, that is more of an implementation detail. They are using the recipes.

In our case the users pick recipes like audit.hcl, project.hcl, etc which are implementing one core piece of GCP infra.

Reply:
Up to you then, I approved already. We can change it if someone opens an issue that it's unclear what recipes are or how they work.